pkg/manager/manager.go

```diff
@@ -70,6 +74,8 @@ func NewFraktiManager(
 	streamingServer streaming.Server,
 	privilegedRuntimeService runtime.RuntimeService,
 	privilegedImageService runtime.ImageService,
+	unikernelRuntimeService runtime.RuntimeService,
+	unikernelImageService runtime.ImageService,
```

Comment:
should make those services params more general? e.g. wrap them under `runtime.RuntimeService` and change params to `(hyperService RuntimeService, streamingServer , priviledgedService RuntimeService, extraRuntimes [] RuntimeService)`